fission/src/ui/components/MainHUD.tsx

```diff
@@ -105,11 +105,11 @@ const MainHUD: React.FC = () => {
                         icon={SynthesisIcons.Gamepad}
                         onClick={() => openModal("change-inputs")}
                     />
-                    <MainHUDButton
+                    {/* <MainHUDButton
```

Comment:
Thinking we should remove this section now.

Reply:
Whoops